doc/README_Template.md

```diff
@@ -0,0 +1,119 @@
+
+YaST &lt;FOO&gt; Module

```

Comment:
We should adapt all READMEs to follow this template, later any repository could be used as an example.